releases/release-1.27/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Kubernetes 1.27 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.27
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.27/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.27/release-team.md)
+* [Meeting Minutes](http://bit.ly/k8s127-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.27 Release Calendar][k8s127-calendar]
+* Contact: [#sig-release] on slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](https://bit.ly/k8s127-enhancements)
```

Comment:
I don't think this link exists yet, also is it correct to call it a "sheet" anymore? 🤔

Reply:
https://bit.ly/k8s127-enhancements now exists :)
But yes we should probably call it the `Enhancements Tracking Board` or `Enhancements Tracking Project Board`